src/wui/CMakeLists.txt

```diff
@@ -1,150 +1,152 @@
 wl_library(wui_sound_options
-  SRCS
-    sound_options.cc
-    sound_options.h
-  DEPENDS
-    base_i18n
-    graphic_styles
-    sound
-    ui_basic
-)
+        SRCS
```

Comment:
Please revert the broken formatting here

Reply:
done